### ISSUE TITLE
TPAC 2022 status report

### ISSUE THREAD

**Issue by LJWatson** [w3c/IntersectionObserver#502]
[TPAC 2022 ](https://www.w3.org/2022/09/TPAC/) is hybrid this year. WebApps will meet in Vancouver on [12 and 13 September](https://www.w3.org/calendar/tpac2022/). Please [register](https://www.w3.org/2022/09/TPAC/registration.html) if you plan to attend in person or remotely.

Editors, we need you to do two things before 30 August 2022:

1. If you want agenda time to discuss your spec(s), please use W3C/webappswg#87 to request it
2. Complete a status report for your spec(s)

#### Status report

Your status report should answer these questions (as a comment on this issue):

1. What progress has your spec made in the last 12 months?
2. Is anything blocking your spec from moving to CR/Rec?
3. If yes, what is your plan to unblock it and do you need any help?

Please post a status report on your spec(s) even if you do not plan to attend TPAC at all.

CC @marcoscaceres and @siusin

**Comment by miketaylr** [comment]
Not an editor, but here's some recent progress:

1. Triaged all issues, and [closed 59](https://github.com/w3c/IntersectionObserver/issues?q=is%3Aissue+commenter%3Amiketaylr+is%3Aclosed) that are no longer relevant.
2. Shipped a [patch version of the polyfill](https://www.npmjs.com/package/intersection-observer/v/0.12.1) on npm, then removed and [archived it](https://github.com/GoogleChromeLabs/intersection-observer).
3. Fixed a few editorial issues.

TODO: need to audit test suite, and identify remaining CR blocker issues (should be very few?).
